### DISCUSSION TITLE
Repo managed by AWS?

### DISCUSSION THREAD

**Discussion by thi-baut** [bottlerocket-os/bottlerocket#2058] [Q&A] [answered]
Hi team, I'm wondering if the repos listed under https://github.com/bottlerocket-os are managed by AWS employee only? I'm concerned about supply chain attacks (malicious code ending up here) and cannot see bottlerocket listed under https://github.com/aws. It's probably a non issue but I prefer to check

**Comment by bcressey** [comment, marked as answer]
The repos under the "bottlerocket-os" org are all managed by AWS employees.
